cirq-google/cirq_google/engine/engine_program.py

```diff
@@ -60,17 +60,15 @@ def __init__(
         self.context = context
         self._program = _program
 
-    # TODO(#6271): Deprecate and remove processor_ids before v1.4
     async def run_sweep_async(
         self,
         job_id: Optional[str] = None,
         params: cirq.Sweepable = None,
         repetitions: int = 1,
-        processor_ids: Sequence[str] = ('xmonsim',),
+        processor_id: str = "xmonsim",
```

Comment:
This default value refers to a now-defunct simulator. Since there's no good default, let's also make this argument required.